src/cards/stats-card.js

```diff
@@ -274,7 +275,11 @@ const renderStatsCard = (stats, options = {}) => {
   STATS.commits = {
     icon: icons.commits,
     label: `${i18n.t("statcard.commits")}${
-      include_all_commits ? "" : ` (${new Date().getFullYear()})`
+      include_all_commits 
+      ? "" 
+      : year 
+        ? ` (${year})`
+        : ` (${new Date().getFullYear()})`
```

Comment:
I think that current behavior with showing current year number when the card actually shows commits count for last year is confusing for users, so i suggest to replace it with `(last year)` as we have in `Contributed to` stat.

```suggestion
      include_all_commits 
      ? "" 
      : year 
        ? ` (${year})`
        : ` (last year)`
```